updates/next.json

```diff
@@ -25,7 +25,7 @@
       "metadata": {
         "rollout": {
           "duration_minutes": 2880,
-          "start_epoch": 1599055200,
+          "start_epoch": 1599645600,
```

Comment:
This one is missing the version update on line 24.

Reply:
Good catch!